reddit/reddit-connector/reddit-connector-1.0.0/org.wso2.carbon.connector/src/test/resources/artifacts/ESB/config/restRequest/reddit/esb_getAnyUserPublicMultisList_mandatory.json

```diff
@@ -0,0 +1,7 @@
+{
+  "apiUrlOauth":"%s(apiUrlOauth)",
+  "accessToken":"%s(accessToken)",
+  "userName":"%s(publicMultiUsername)",
+  "expandSrs":"%s(expandSrs)"
+}
+
```

Comment:
Fixed in https://github.com/wso2-attic/esb-connectors/commit/c61da2ff6de0ced270ed9698ed4fbd8227a92f5b